src/UI/Areas/Identity/Pages/Account/Manage/ExternalLogins.cshtml.cs

```diff
@@ -57,7 +58,9 @@ internal class ExternalLoginsModel<TUser> : ExternalLoginsModel where TUser : Id
             OtherLogins = (await _signInManager.GetExternalAuthenticationSchemesAsync())
                 .Where(auth => CurrentLogins.All(ul => auth.Name != ul.LoginProvider))
                 .ToList();
-            ShowRemoveButton = user.PasswordHash != null || CurrentLogins.Count > 1;
+
+            string passwordHash = null;
```

Comment:
I couldn't find a way to get this value. I tried IUserPasswordStore but that service seems to be not necessarily registered. @HaoK any ideas?

Reply:
You can use the IUserStore service and cast it to IUserPasswordStore